vertx-web/src/main/java/io/vertx/ext/web/handler/impl/BodyHandlerImpl.java

```diff
@@ -80,9 +78,11 @@ public BodyHandler setMergeFormAttributes(boolean mergeFormAttributes) {
     AtomicInteger uploadCount = new AtomicInteger();
     boolean ended;
 
-    private BHandler(RoutingContext context) {
+    public BHandler(RoutingContext context) {
```

Comment:
This is to avoid synthetic accessor code that the Java compiler would need to create to access the constructor from the outer class, and effectively avoid a "boxing" operation during each request. The BHandler class was and still is private, and is not able to be used outside the BodyHandlerImpl.